include/yaramod/utils/utils.h

```diff
@@ -43,14 +45,22 @@ bool startsWith(const std::string& str, const T& withWhat)
  * @return Number converted to string.
  */
 template <typename T>
-std::string numToStr(const T num, std::ios_base &(*format)(std::ios_base&) = std::dec, bool showbase = false)
+std::string numToStr(const T num, std::ios_base &(*format)(std::ios_base&) = std::dec, bool showbase = false, bool toUpper = false)
 {
 	std::ostringstream os;
 	if (showbase)
 		os << format << std::showbase << num;
 	else
 		os << format << num;
-	return os.str();
+	if (!toUpper)
```

Comment:
There is already stream manipulator for making this uppercase https://en.cppreference.com/w/cpp/io/manip/uppercase